x/interchainquery/keeper/abci.go

```diff
@@ -54,7 +55,7 @@ func (k Keeper) EndBlocker(ctx sdk.Context) {
 			// query was removed; delete datapoint
 			k.DeleteDatapoint(ctx, dp.Id)
 		} else {
-			if dp.LocalHeight.Int64()+int64(q.Ttl) > ctx.BlockHeader().Height {
+			if dp.LocalHeight.Int64()+cast.ToInt64(q.Ttl) > ctx.BlockHeader().Height {
```

Comment:
Nit: looks like we're now using casts where we previously were using `int64(X)` but not where we are using `X.Int64()`. Is your thinking that in the second case, perhaps the type has a custom `Int64()` conversion method?

Reply:
I just fixed what was flagged by gosec

It looks like `Int64.(...)` is a method defined on `sdk.Int`, so I imagine it's safe!